### PR TITLE
Player list

### LINKED ISSUE #22: Remove Friend
As a user, I should be have the ability to remove a user from my friend list

Acceptance Criteria:
The app needs a "X" (remove button) on players within the Friend module on the Dash. When clicking this button, the player is removed from friendList and the ability to add them again is restored to the community list.

### LINKED ISSUE #23: Add User To Friends
As a user, I should be have the ability to add a user in the community to my friends list

Acceptance Criteria: 
User should be able to view (on large screens) or navigate (on mobile) to view the list of people registered on the app.  User should be able to click a button to send a request to another user to become friends

### PR DESCRIPTION
**Pull request by tysnj**
## Is this a fix or a feature?
Feature

## What is the change?
Introduction of PlayerList component which displays community/friends in a sidebar on desktop, and a separate page on smaller screensizes (<1024)

## What does it fix?
Users can add and remove friends from their relationships.

## Where should the reviewer start?
Fetch and checkout the playerList branch

## How should this be tested?
Add friend; remove friend; confirm responsiveness; confirm redirect

closes #39 
closes #22
closes #23

### CODE REVIEW

File: src/APICalls/APICalls.js [hunk 1]
```diff
@@ -1,5 +1,14 @@
+const endpoints = {
+  playersProd: 'https://fore-finder-be.herokuapp.com/api/v1/players',
+  playersDev: 'http://8f05812ea9bf.ngrok.io/api/v1/players',
+  coursesProd: 'https://fore-finder-be.herokuapp.com/api/v1/courses',
+  coursesDev: 'http://8f05812ea9bf.ngrok.io/api/v1/courses',
+  eventsProd: 'https://fore-finder-be.herokuapp.com/api/v1/events',
+  eventsDev: 'http://8f05812ea9bf.ngrok.io/api/v1/events'
+}
```
Comment (errabun) [inline src/APICalls/APICalls.js:8]: This is a great idea to change this as the endpoints are constantly changing with ngrok

File: src/APICalls/APICalls.js [hunk 2]
```diff
@@ -20,6 +29,17 @@ export const getAllCourses = () => {
...
 
+export const getAllEvents = () => {
+  return fetch(endpoints.eventsDev)
+    .then(resp => {
+      if (!resp.ok) {
+        throw new Error('Can\'t fetch any events, please try again!')
+      } else {
+        return resp.json()
+      }
+    })
```
Comment (errabun) [inline src/APICalls/APICalls.js:40]: We still need to add a catch for these fetches

Comment (tysnj) [reply src/APICalls/APICalls.js:40]: nice _catch_

File: src/APICalls/APICalls.js (hunk 1 above)
Comment (tysnj) [reply src/APICalls/APICalls.js:8]: @jon-schlandt deserves the credit for that one

File: src/components/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -29,43 +30,55 @@ const Dashboard = ({ events, currentUserId, screenWidth, handleInviteAction }) =
...
-      {teeTimeType === 'committed'
-        ? (screenWidth < 768) && 
-          <TeeTimeContainer 
-            title='Committed Tee Times' events={committedTeeTimes} windowWidth={screenWidth} handleInviteAction={handleInviteAction}
-          />
-        : (screenWidth < 768) && 
-          <TeeTimeContainer 
-            title='Available Tee Times' events={availableTeeTimes} windowWidth={screenWidth} handleInviteAction={handleInviteAction}
+
+      {screenWidth >= 1024 &&
```
Comment (errabun) [inline src/components/Dashboard/Dashboard.js:34]: This is great logic for changing if the view of the aside base on screen size!

**Review by errabun — COMMENTED** [review]
Not sure what else we can display for the friends in the aside.  Love how it shows add/remove friends when you are in the community!

**Pull request merged**